cmd/satellite/Dockerfile

```diff
@@ -12,6 +12,9 @@ COPY web/satellite/ /app
 RUN npm install
 RUN npm run build
 
+FROM alpine as ca-cert
```

Comment:
Should we add this to other Dockerfiles as well?

Reply:
Only if the other container images need to verify the TLS cert chain of sites that don't use Let's Encrypt. I don't think the other images do. This one does because it reaches out to a mailserver that doesn't use Let's Encrypt.